<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>–≠–ª–µ–∫—Ç—Ä–æ–Ω–Ω—ã–π –î–Ω–µ–≤–Ω–∏–∫</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="rBtn">
        <property name="text">
         <string>—Ä—É–∫–æ–≤–æ–¥—Å—Ç–≤–æ –ø–æ –∏—Å–ø–æ–ª—å–∑–æ–≤–∞–Ω–∏—é</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="off">
        <property name="text">
         <string>–ó–∞–≤–µ—Ä—à–µ–Ω–∏–µ —Ä–∞–±–æ—Ç—ã</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SetBtn">
        <property name="maximumSize">
         <size>
          <width>31</width>
          <height>31</height>
         </size>
        </property>
        <property name="text">
         <string>üîß</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>–ì–ª–∞–≤–Ω–∞—è —Å—Ç—Ä–∞–Ω–∏—Ü–∞</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>–î–æ–±–∞–≤–∏—Ç—å —É—á–µ–Ω–∏–∫–∞</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>201</width>
          <height>124</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>–£–¥–∞–ª–∏—Ç—å —É—á–µ–Ω–∏–∫–∞ –ø–æ</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox">
             <item>
              <property name="text">
               <string>–§.–ò.–û</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>–ù–æ–º–µ—Ä—É</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="delLine"/>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>–ü—Ä–∏—á–∏–Ω–∞</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="prichLine"/>
         </item>
         <item>
          <widget class="QPushButton" name="delbut">
           <property name="text">
            <string>–£–¥–∞–ª–∏—Ç—å</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>201</width>
          <height>133</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>–§.–ò.–û</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>–ù–æ–º–µ—Ä</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>–ö—É—Ä—Å</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLineEdit" name="fio"/>
             </item>
             <item>
              <widget class="QLineEdit" name="nymber"/>
             </item>
             <item>
              <widget class="QLineEdit" name="course"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="dobBtn">
           <property name="text">
            <string>–î–æ–±–∞–≤–∏—Ç—å</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>30</y>
          <width>439</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QRadioButton" name="radioButton">
             <property name="text">
              <string>–ü–æ—Å—Ç–∞–≤–∏—Ç—å –∑–∞—á–µ—Ç</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>—É—á–µ–Ω–∏–∫—É –ø–æ</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_2">
             <item>
              <property name="text">
               <string>–§.–ò.–û</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>–ù–æ–º–µ—Ä—É</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="indexLine"/>
           </item>
           <item>
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>–∑–∞</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_3">
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>—Ç—Ä–∏–º–µ—Å—Ç—Ä.</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>–ü–æ –ø—Ä–µ–¥–º–µ—Ç—É</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_4">
               <item>
                <property name="text">
                 <string>–§–∏–∑–∏–∫–∞</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>–ú–∞—Ç–µ–º–∞—Ç–∏–∫–∞</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>–ò–Ω—Ñ–æ—Ä–º–∞—Ç–∏–∫–∞</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="zachBtn">
             <property name="text">
              <string>–í—ã—Å—Ç–∞–≤–∏—Ç—å</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>–ñ—É—Ä–Ω–∞–ª</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QTableWidget" name="tableWidget"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>1—Ç—Ä–∏–º</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <widget class="QTableWidget" name="tableWidget_2"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>2—Ç—Ä–∏–º</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QTableWidget" name="tableWidget_3"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>3—Ç—Ä–∏–º</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QTableWidget" name="tableWidget_4"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>–ó–∞–≥—Ä—É–∑–∏—Ç—å</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>–°–æ—Ö—Ä–∞–Ω–∏—Ç—å</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
